README.md

```diff
@@ -87,24 +84,23 @@ schemas dynamically, taking request selectors into account.
 Unless you have specific needs not covered by this mechanism, there's no need to implement your
 own `SchemaProvider` services.
 
-## SlingDataFetcher selection with Schema Directives
-
-The GraphQL schemas used by this module can be enhanced using
-[schema directives](http://spec.graphql.org/June2018/#sec-Language.Directives)
-(see also the [Apollo docs](https://www.apollographql.com/docs/graphql-tools/schema-directives/) for how those work)
-that select specific `SlingDataFetcher` services to return the appropriate data.
-
-A default data fetcher is used for types and fields which have no such directive.
+## SlingDataFetcher selection using the `@fetcher` directive
```

Comment:
I would say "built-in @fetcher directive" and use this "built-in" term for the other directives as well. And we can then add a short "built-in directives" section which explains the concept.